src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
```suggestion
  }
  
  if (days >= 3) {
```

you don't need this extra check. you already checked this in first if and return(stop function) if `days >= 7`.